### DISCUSSION TITLE
123云盘alist上面挂载打不开：failed get objs: failed to list objs: 123pan-global-slb forbidden client ip

### DISCUSSION THREAD

**Discussion by MrDian13221202995** [AlistGo/alist#6023] [Q&A]
我向123云盘的群聊里反馈了这个问题给我了解决方法：
![微信图片_20240211100108](https://github.com/alist-org/alist/assets/107745425/1dd061ca-9d11-48d5-a474-f523d3f4e811)
我还是很想用alist，这个合并代码修复这个我也不懂，懂得帮帮我，我想一直用的，我还需要过段时间才能用，想着尽快的用上alist

**Comment by anwen-anyi** [comment]
着急就自己下载：https://github.com/alist-org/alist/actions/runs/7840585181

**Reply by MrDian13221202995** [reply]
![image](https://github.com/alist-org/alist/assets/107745425/27377fd3-25a1-4c52-94a1-864668699e24)
z这也不行啊

**Reply by anwen-anyi** [reply]
无任何问题，本来就是提交的后端，和前端没啥关系，而且这里面有编译好的 自己下载更新就好 :joy_cat: 
![image](https://github.com/alist-org/alist/assets/56105412/aad8371e-c04c-495e-85df-18fb176aeaf5)

**Reply by anwen-anyi** [reply]
自己下载这个吖
https://github.com/alist-org/alist/actions/runs/7840585181/artifacts/1232556507
![image](https://github.com/alist-org/alist/assets/56105412/0036fbb5-83c7-476b-b225-e33e08e79cbe)

**Comment by AspirantH** [comment]
123pan官方不是不让用了吗？

**Comment by MrDian13221202995** [comment]
![微信图片_20240215200358](https://github.com/alist-org/alist/assets/107745425/883aaa30-05b3-4da0-8223-264308b43551)
你在哪里看的不能用，既然他们有心联系alist的作者来修复bug，那么就肯定是可以用的，别造谣言

**Reply by AspirantH** [reply]
有那么一阵子alist的确用不了123pan
https://github.com/alist-org/alist/issues/5513

**Reply by MrDian13221202995** [reply]
那么那是用不了，还是官方说明不能用

**Reply by AspirantH** [reply]
用不了是因为官方屏蔽了，官方为什么要给说明，123官方的任何变动以他的角度也没必要发公告说明，这不是一种强制性约束，发公告和哪怕没发公告但事实上屏蔽了，结果都是用不了，这是我的判断，当然你也有你自己的判断

**Reply by MrDian13221202995** [reply]
说实在我有经常在用ALIST我不觉得有什么不好的，你说的那一阵子我没有判断，况且我一直是在能用的状态下，我现在也不用判断，既然他们能说协助alist作者完善123云盘的挂载储存，那是可以用的

**Reply by AspirantH** [reply]
我也希望能用，毕竟多一个选择，可是你现在也是遇到无法挂载的问题才发的这个帖子啊，字面意思是你的IP被官方封了，那就是触发了官方的机制呗，也许官方就是不想让我们用alist挂载呢？我就是因为（可能因）官方反复针对alist，现在干脆不用123了

**Reply by MrDian13221202995** [reply]
我没有遇到无法挂载的问题，这个只是个别文件打不开，之后在123网站里打不开文件页，仅仅重启光猫的手段就可以解封，触发他们的机制是他们的一个bug，他们不是说了试图联系alist作者，只是一直联系不上，况且在此之上，如果我的账号封了什么的，他们主动帮我解封，有这样的热情，何来针对alist？

**Reply by AspirantH** [reply]
官方用户协议明确说了，使用第三方软件或未授权软件是不被支持的

5.4.您承诺不会利用本服务进行任何违法或不当的活动，包括但不限于下列行为：
5.4.9.通过非123云盘公司开发、授权的第三方软件、插件、外挂、系统，登录或使用123云盘服务或为任何用户自动登录到123云盘账号代理身份验证凭据，或制作、发布、传播上述工具；

**Reply by AspirantH** [reply]
但我不确定alist是否取得授权

**Reply by AspirantH** [reply]
我说的针对意思是他们通过特定措施妨碍alist正常挂载123pan

**Reply by MrDian13221202995** [reply]
我真的是服了，既然不授权，干嘛联系alist的作者跟他们合作修复bug

**Reply by MrDian13221202995** [reply]
允许用，但他们的意思还是要到下一个正式版，这样不容易就这么封禁了，你看看123他增加了离线，增加了TV版，功能一点点的增加起来，这个是对123的账号一个安全作用
